BezierSwift/Sources/StandardComponent/Spinner/BezierSpinner.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  BezierSpinner.swift
+//
+//
+//  Created by Tom on 6/29/24.
+//
+
+import SwiftUI
+
+private enum Constant {
+  static let delayPoint: Double = 0.7
+  static let maxFillPoint: Double = 1.0
+  static let animationDuration: Double = 2.2
+  static let fullCircle: Double = 360
+  static let minAngle: Double = 1
+  static let maxAngle: Double = 359
+}
+
+public struct BezierSpinner: View, Themeable {
+  @Environment(\.colorScheme) public var colorScheme
+  @State private var fillPoint = 0.0
+  @State private var isRotating = false
+  
+  private let configuration: BezierSpinnerConfiguration
+  
+  public init(configuration: BezierSpinnerConfiguration) {
+    self.configuration = configuration
+  }
+  
+  public var body: some View {
+    ZStack {
+      Track(configuration: self.configuration)
+        .stroke(self.palette(self.configuration.trackColor), lineWidth: configuration.lineWidth)
```

Comment:
```suggestion
        .stroke(self.palette(self.configuration.trackColor), lineWidth: self.configuration.lineWidth)
```

요거 외에도 self. 을 붙인 부분과 안붙인 부분이 섞여있어 통일해야 할 것 같아요